lib/request.ts

```diff
@@ -0,0 +1,90 @@
+import merge from "lodash.merge";
+
+const DEFAULT_HEADERS = {
+  "Content-Type": "application/json",
+};
+
+const _get = async (url, params, { headers = null }) => {
+  const res = await fetch(`${url}`, {
+    method: "GET",
+    headers: headers,
+  });
+
+  if (!res.ok) {
+    let message;
+    // status = res.status
+    // try message = res.json || res.text || res.body.message
```

Comment:
Before the final merge, we should remove the commented out code or make use of it. :)